quantities/concepts.hpp

```diff
@@ -14,13 +14,50 @@ namespace internal {
 using namespace base::_traits;
 using namespace quantities::_quantities;
 
+template<typename G>
+concept additive_group = requires(G x, G y) {
+  G{};
+  { -x } -> std::same_as<G>;
```

Comment:
Should we have `+x`?

Also wondering if we should require that `G::zero` exist.  That would sometimes be useful.

Reply:
Re zero, that is what G{} is for. double::zero certainly does not exist.